### ISSUE TITLE
command summarizePage add  auto Splitter content

### ISSUE THREAD

**Issue by chenpx976** [josStorer/chatGPTBox#251]
**Is your feature request related to a problem? Please describe.**
**新功能是否与解决某个问题相关, 请描述**

command summarizePage add  auto Splitter content

**Describe the solution you'd like**
**你期望的新功能实现方案**

prompt

```
You are LongTextAnalyzerGPT.
I will submit you a long text divided in N parts. 
Each part will start by Part X. After each part I submit, ask me for the next part. Don't do any analysis before all the parts are submitted.
```

**Additional context**
**其他**

**Comment by josStorer** [comment]
This is usually meaningless, because the current submitted content is already controlled to the context length limit of the GPT-3.5 API. Using Langchain to build a custom backend will be more helpful in solving this problem

**Comment by chenpx976** [comment]
You are right, I am actually using ChatGPT Web, can this feature be used for web services?

**Comment by Maddaxman** [comment]
I tried to use prompts like this in the 3.5 model to input summaries of long texts in segments, but it was not actually feasible . lol . After three or four segments, I had already forgotten the first one. Even when inputting the third segment, it started creating new content based on the content of that segment.

**Comment by chenpx976** [comment]
@Maddaxman 
I saw another project that uses GPT-3.5 to handle extremely long texts by split long texts
And I have personally used another Chrome extension that is effective
extension: https://chrome.google.com/webstore/detail/split-long-text-for-chat/ninfbacppokhhaekjbgfhemhmlalfmmn
repo: https://github.com/easychen/gpt-bat 
demo: https://gpt.level06.com/

**Comment by josStorer** [comment]
ok, I think your demand should be the same as this issue #123
